lib/perl/Genome/Command/DelegatesToResult.pm

```diff
@@ -15,13 +15,21 @@ class Genome::Command::DelegatesToResult {
         },
     ],
     has_optional_input => [
+        sponsor => {
+            is => 'UR::Object',
+            doc => 'The responsible party for this result (defaults to the current user)',
+        },
+        requestor => {
```

Comment:
This is a conceit to the fact that the `requestor` is indirectly specified by the VariantReporting framework.  Those commands have an explicit process_id supplied which is then indirectly translated to the sponsor.  Ptero evaluates the presence (or not) of the input without first loading the Perl class, so it doesn't realize this relationship exists. Thus it bails out even though the value will actually be present when the Command is created.